kimchi/src/bench.rs

```diff
@@ -77,9 +77,14 @@ impl BenchmarkCtx {
     }
 
     /// Produces a proof
-    pub fn create_proof(&self) -> (ProverProof<Vesta, OpeningProof<Vesta>>, Vec<Fp>) {
+    pub fn create_proof(
+        &self,
+    ) -> (
+        ProverProof<Vesta, OpeningProof<Vesta>, KIMCHI_COLS>,
```

Comment:
You can omit this argument.

Reply:
Oh this is true, now it is 15 by default 👍🏻